C/CLFFT/build_tarballs.jl

```diff
@@ -0,0 +1,42 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "CLFFT"
+version = v"2.12.2"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/clMathLibraries/clFFT.git", "1e4833f060976971c4df4b54b1b9ad1620aaf1fb")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+
+install_license ./clFFT/LICENSE
+
+cmake -DCMAKE_INSTALL_PREFIX=${prefix} -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} -DCMAKE_BUILD_TYPE=Release -S ./clFFT/src -B ./clFFT/build
+cmake --build ./clFFT/build --target install -j${nproc}
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = supported_platforms()
+
+
+# The products that we will ensure are always built
+products = [
+    LibraryProduct("libclFFT", :libclfft)
+]
+
+# Dependencies that must be installed before this package can be built
+dependencies = [
+    Dependency(PackageSpec(name="OpenCL_Headers_jll", uuid="a7aa756b-2b7f-562a-9e9d-e94076c5c8ee"))
```

Comment:
This must be, again, a build dependency like https://github.com/JuliaPackaging/Yggdrasil/blob/442f2006db23f3c2ed78c16a7b8f8724d5f403ca/O/OpenCL/build_tarballs.jl#L34 Header files are ***ALWAYS*** build dependencies only, you can't do anything with that at runtime.  And you likely want to specify the version you want to use.